src/index.js

```diff
@@ -80,10 +80,6 @@ app.use((err, req, res, next) => {
     }
   });
 });
-// routes
-app.use(routes);
-app.get('/', (req, res) => res.status(200).send({ message: 'welcome to BN: jubilee-team' }));
-app.all('*', (req, res) => res.send({ message: 'route not found' }));
 
```

Comment:
Why are we removing this?

Reply:
Because they are redundant code. 
- The line `app.use(routes);` is part of an outdated implementation routes are now with the line `app.use('/api', routes);` on this page. 
- The line `app.get('/', (req, res) => res.status(200).send({ message: 'welcome to BN: jubilee-team' }));` isn't a functional aspect of the requirement for this project and is not in use.
- All requests to endpoints that can not be found on the server have been elegantly handled by the following middlewares 
```
app.use((req, res, next) => {
  const err = new Error('Not Found');
  err.status = 404;
  next(err);
});
if (!isProduction) {
  // eslint-disable-next-line no-unused-vars
  app.use((err, req, res, next) => {
    // eslint-disable-next-line no-console
    console.log(err.stack);

    res.status(err.status || 500);

    res.json({
      errors: {
        message: err.message,
        error: err
      }
    });
  });
}
```
Making the line `app.all('*', (req, res) => res.send({ message: 'route not found' }));` of no use.

Reply:
Alright, cool. Good to know that this is all covered in the current implementation